### DISCUSSION TITLE
Delayed Education Benefits

### DISCUSSION THREAD

**Discussion by NimeshHB** [community/community#129557] [GitHub Education]
### Hi everyone,

I have been approved for education benefits but have not received them within the expected timeframe of 5 days. I am seeking assistance to resolve this issue.

**Comment by davevad93** [comment]
Hi @NimeshHB ,

Look at this discussion #128457 , this user already had the benefits available and the `PRO` badge and still had to wait almost 10 days to finally access his benefits.
